recipes/aws-kvs-pic/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+import os
+from conans import ConanFile, tools, CMake
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+class awskvspicConan(ConanFile):
+    name = "aws-kvs-pic"
+    license = "Apache-2.0"
+    homepage = "https://github.com/awslabs/amazon-kinesis-video-streams-pic"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = ("Platform Independent Code for Amazon Kinesis Video Streams")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {'shared': False, "fPIC": True}
+    generators = "cmake"
+    topics = ("conan", "aws", "kvs", "stream")
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if not self._cmake:
+            self._cmake = CMake(self)
+            self._cmake.definitions["BUILD_DEPENDENCIES"] = False
+            self._cmake.configure()
+        return self._cmake
+
+    def validate(self):
+        if (self.settings.os == "Macos" and self.options.shared):
```

Comment:
Until a couple of commits ago it was only possible to build `static`. Shared has been enabled but `apple-clang` errors out during linking when shared. 
I'll report this on the main repo.

Reply:
Actually it fails on Windows + shared as well. I opened a bug report.